### PR TITLE
Some Atum QOL suggestions

### PR DESCRIPTION
**Pull request by FieldArmada**
In commit order, feel free to take none or any of these:

* Silk touch is borderline a downside in Atum because you can't process Bone, Relic, or Khnumite. Add smelting recipes for bone (usually drops 1-3) and khnumite (usually drops 2-4). I don't know what to do about Relic Ore; I don't think there's a sensible way to average that loot table.
* Atum gold / iron tools and armor are now recyclable like their vanilla counterparts. Makes the bandit and chest trash slightly less annoying at least.
* Add `forge:ores` to Bone Ore and Relic Ore so that they scan properly. Also make Bone Ore scan as common.
* Flax seed is, unlike every other seed, not compostable.
* `Quandry` -> `Quandary`.
* Add the dye QOL that exists for normal wool and carpet to linen and linen carpet. I was thinking about doing this for ceramic and crystal glass too, but it seems like there's already something in place for ceramic and there are so many versions of glass.

### CODE REVIEW

**Comment by MuteTiefling** [comment]
> I don't know what to do about Relic Ore; I don't think there's a sensible way to average that loot table.

It should probably just melt down to gold ingots, to be honest ;D

File: kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/compote/composting.js [hunk 1]
```diff
@@ -24,7 +24,9 @@ onEvent('recipes', (event) => {
                 { item: 'sushigocrafting:wasabi_root', chance: 0.65 },
                 { item: 'sushigocrafting:soy_bean', chance: 0.65 },
                 { item: 'sushigocrafting:cucumber', chance: 0.65 },
-                { item: 'sushigocrafting:rice', chance: 0.65 }
+                { item: 'sushigocrafting:rice', chance: 0.65 },
+
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/compote/composting.js:28]: Found a wild newline😄

**Review by NillerMedDild — APPROVED** [review]
Nice changes you've made, thank you very much for contributing :D

I think Relic Ore -> Gold Ingot works, would you mind adding that to this PR?

File: kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/compote/composting.js (hunk 1 above)
Comment (NillerMedDild) [reply kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/compote/composting.js:28]: Oh nevermind, I see it follows the convention of that file ^^'

**Comment by NillerMedDild** [comment]
Nice, thanks!

**Pull request merged**